Source/Particles/ParticleCreation/CopyParticle.H

```diff
@@ -0,0 +1,90 @@
+#ifndef COPYPARTICLE_H_
+#define COPYPARTICLE_H_
+
+#include "WarpXParticleContainer.H"
+
+/**
+ * \brief Functor to copy one particle
+ *
+ * This is meant to be a very small class stored in managed memory,
```

Comment:
The CopyParticle functor itself doesn't have to be in managed memory, does it? It is a small struct that stores pointers to managed memory, but the functor itself can be copied into kernels by value.

Reply:
Oh that's correct, I'll change that, thanks!

Reply:
Done and tested on CPU and GPU.